### PR TITLE
RequestCollector: Show available view variables

### PR DESCRIPTION
**Pull request by snapshotpl**
I have extend and finish #151
Changes since @ThaDafinser version:
* variable's types,
* remove duplicate code,
* fixes and improvements in view,
* recursive search ViewModels,

### CODE REVIEW

File: view/zend-developer-tools/toolbar/request.phtml [hunk 1]
```diff
@@ -41,6 +40,12 @@ $color = ($this->collector->getStatusCode() < 400)
             <span class="zdt-detail-label">Route</span>
             <span class="zdt-detail-value"><?php echo $this->collector->getRouteName(); ?></span>
         </span>
-        <?php echo $this->ZendDeveloperToolsDetailArray('Templates', $this->collector->getTemplateNames()); ?>
+        <?php foreach($this->collector->getViews() as $value): ?>
```
Comment (Ocramius) [inline view/zend-developer-tools/toolbar/request.phtml]: Thoughts on using the symfony vardumper component for this sort of stuff?

**Comment by ThaDafinser** [comment]
@snapshotpl nice work!

File: view/zend-developer-tools/toolbar/request.phtml (hunk 1 above)
Comment (snapshotpl) [inline view/zend-developer-tools/toolbar/request.phtml]: SF dumper will be good for printing config, but in this case... I don't think so.

Comment (snapshotpl) [inline view/zend-developer-tools/toolbar/request.phtml]: @Ocramius what about this?

**Comment by Ocramius** [comment]
Looks good enough: merging :-)

**Pull request merged**